ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master__okd-scos.yaml

```diff
@@ -26,17 +26,12 @@ images:
   to: cluster-samples-operator
 promotion:
   to:
-  - name: "4.17"
+  - name: scos-4.17
     namespace: origin
 releases:
-  initial:
-    integration:
-      name: "4.17"
-      namespace: origin
   latest:
     integration:
-      include_built_images: true
-      name: "4.17"
+      name: scos-4.17
       namespace: origin
```

Comment:
this will add the created image to the scos-4.17 imagestream. To mirror to quay you will have to add it in the mapping below.

Reply:
Thanks, this seems to have worked! 🥇